src/hypothesis/settings.py

```diff
@@ -198,6 +201,10 @@ def define_setting(cls, name, description, default, options=None):
         all_settings[name] = Setting(
             name, description.strip(), default, options)
         setattr(cls, name, SettingsProperty(name))
+        if cls.default:
+            setattr(cls.default, name, default)
+        for profile in cls._profiles.values():
```

Comment:
Ok so I added a test 
```
def test_define_setting_then_loading_profile():
    x = Settings()
    Settings.define_setting(
        u'fun_times',
        default=3, description=u'Something something spoon',
        options=(1, 2, 3, 4),
    )
    Settings.register_profile('hi', Settings(fun_times=2))
    assert x.fun_times == 3
    assert Settings.get_profile('hi').fun_times == 2
```

So I make a settings object `x`

I define a new setting 

x gets the default value while any new profiles get the value I set. Seems right to me. Though this PR is the lat thing I do before bed so I would not be surprised if i'm just being dense